Sources/KituraNIO/HTTP/SSLConfiguration.swift

```diff
@@ -20,26 +20,46 @@ import SSLService
 /// A helper class to bridge betweem SSLService.Configuration (used by Kitura) and TLSConfiguration required by NIOOpenSSL
 internal class SSLConfiguration {
    
-    private var certificateFilePath: String? = nil
+    private var certificateFilePath: String?
     
-    private var keyFilePath: String? = nil
+    private var keyFilePath: String?
+
+    private var certificateChainFilePath: String?
+
+    private var password: String?
     
     // TODO: Consider other TLSConfiguration options (cipherSuites, trustRoots, applicationProtocols, etc..)
 
     /// Initialize using SSLService.Configuration
     init(sslConfig: SSLService.Configuration) {
         self.certificateFilePath = sslConfig.certificateFilePath
         self.keyFilePath = sslConfig.keyFilePath
+        self.certificateChainFilePath = sslConfig.certificateChainFilePath
+        self.password = sslConfig.password
     }
 
     /// Convert SSLService.Configuration to NIOOpenSSL.TLSConfiguration
     func tlsServerConfig() -> TLSConfiguration? {
             // TODO: Consider other configuration options
-            // TODO: Add support for PKCS#12-formatted certificates
             if let certificateFilePath = certificateFilePath, let keyFilePath = keyFilePath {
                 return TLSConfiguration.forServer(certificateChain: [.file(certificateFilePath)], privateKey: .file(keyFilePath))
             } else {
-                return nil
+                /// TLSConfiguration for PKCS#12 formatted certificate
+                if let certificateChainFilePath = certificateChainFilePath, let password = password {
```

Comment:
@nethraravindran Looks like a `guard let` statement will fit better here? What do you say?

Reply:
Yes :) Thanks @pushkarnk will make the change.